test/scripts/r2.js

```diff
@@ -0,0 +1,54 @@
+import { Response } from '@miniflare/core'
+
+// Temporary until Miniflare supports R2
+export function createR2Bucket () {
+  const bucket = new Map()
+
+  return {
+    put: async (key, value, putOpts = {}) => {
+      bucket.set(key, {
+        body: value,
+        httpMetadata: putOpts.httpMetadata || {},
+        customMetadata: putOpts.customMetadata || {}
+      })
+
+      return Promise.resolve({
+        httpMetadata: putOpts.httpMetadata,
+        customMetadata: putOpts.customMetadata,
+        size: value.length
+      })
+    },
+    get: async (key) => {
+      const value = bucket.get(key)
+      if (!value) {
+        return undefined
+      }
+
+      const response = new Response(value.body, { status: 200 })
```

Comment:
Should we be setting content-type headers ?

Reply:
Probably yes! On actual block get response and not R2 response, right?